tools/typescript2fable.js

```diff
@@ -497,7 +536,8 @@ function getModule(name) {
       interfaces: [],
       properties: [],
       methods: [],
-      modules: []
+      modules: [],
+      enums: []
```

Comment:
We need to add a check for `enums` length in https://github.com/fsprojects/Fable/blob/master/tools/typescript2fable.js#L577. Or maybe we can do something generic like checking at least one array property (`Array.isArray(mod[k])`) satisfies `length > 0`.